cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -441,6 +446,10 @@ instance ( ConvertRawHash blk
         ChainDB.ScheduledGC slot _difft ->
           "Scheduled a garbage collection for " <> condenseT slot
       ChainDB.TraceOpenEvent ev -> case ev of
+        ChainDB.StartedOpeningDB -> "Started opening DB"
+        ChainDB.StartedOpeningImmutableDB -> "Started opening Immutable DB"
```

Comment:
I wanted to check that the casing is ok and consistent with https://github.com/input-output-hk/cardano-node/pull/3387.

Reply:
@newhoggy, it'd be nice to disambiguate `DB`, I think -- calling it `Chain DB` might be more more usefully specific.

Reply:
I've reworded "Started opening Immutable DB" to "Started opening Chain DB"

Reply:
@newhoggy, what I meant was the message saying `Started opening DB` was a little ambiguous.

Reply:
Cool.  Fixed!